dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -830,6 +830,22 @@ public void rename(String src, String dst, RenamePOptions options)
     }
 
     try {
+      boolean overWrite = options.hasS3SyntaxOptions()
+          && options.getS3SyntaxOptions().hasOverwrite()
+          && options.getS3SyntaxOptions().getOverwrite();
+      if (dstUfs.exists(dst)) {
```

Comment:
```suggestion
      if (options.hasS3SyntaxOptions() && options.getS3SyntaxOptions().hasOverwrite()) {
          if (dstUfs.exists(dst)) {
              if (!overwrite) {
                  ....
              } else {
                  ...
              }
          }
```

Reply:
only if the request has the overwrite option, it will execute the part logic of overwrite